.github/workflows/github-actions.yml

```diff
@@ -1,26 +1,12 @@
-name: assignment-test
-on: 
-    push:
-        tags-ignore:
-            - '*'
-        branches:
-            - '*'
+name: assignment-autotest-test
+on: [push]
 jobs:
-    unit-test:
+    test:
         container: cuaesd/aesd-autotest:unit-test
-        runs-on: self-hosted
+        runs-on: ubuntu-latest
```

Comment:
@ritikar97 please use self hosted runners as described in the assignment, and use the version of this github actions file in the assignments-base repository